<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>899</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="Codificacao">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Codificacao</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="entrada">
          <property name="statusTip">
           <string extracomment="Escreva a frase aqui"/>
          </property>
          <property name="placeholderText">
           <string>Escreva sua frase aqui</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="botaoCodificar">
          <property name="text">
           <string>Codificar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="text">
           <string>Leitura do arquivo</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="criarArvore">
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="text">
           <string>Dicionário</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="Linha1"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="text">
           <string>Árvore</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="tree">
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_4">
          <property name="text">
           <string>Texto Codificado</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="textoCodificado">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="plainText">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Decodificacao</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="entrada2">
          <property name="placeholderText">
           <string>Escreva a frase em huffman aqui</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="botaoDecodificar">
          <property name="text">
           <string>Decodificar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_6">
          <property name="text">
           <string>Árvore de Huffman</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="novaArvore"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_5">
          <property name="text">
           <string>Frase original</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="fraseOriginal">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>666</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>botaoCodificar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
